deploy/charts/kubernetes-cluster-prep/README.md

```diff
@@ -0,0 +1,61 @@
+# Helm chart for Conjur Kubernetes cluster prep
```

Comment:
NICE README.md!!!!!  I'll look at this in more depth after reviewing the other files.

Reply:
We'll need a table with all configurable settings.
There's an example in cyberark/conjur-oss-helm-chart/conjur-oss/README.txt.